week3/TaskOOP/src/com/JH/SavingAccount.java

```diff
@@ -0,0 +1,18 @@
+package com.JH;
+
+public class SavingAccount extends Account{
+
+    private double interestRate;
+
+    public SavingAccount(int accountNumber, String accountHolderName, double balance, double interestRate) {
+        super(accountNumber, accountHolderName, balance);
+        this.interestRate = interestRate;
+    }
+
+    public void calculateInterest() {
+        double interestAmount = getBalance() * (interestRate/100);
```

Comment:
make sure that the equation does't skip double values as (interestRate/100) -> int/int = int
so , cast one of the numbers to double or make it like that `(interestRate/100.00) `

Reply:
ok 👌